package.json

```diff
@@ -33,7 +33,7 @@
 		"zodcli": "^0.0.4"
 	},
 	"peerDependencies": {
-		"vercel": "^28.0.2",
+		"next-on-pages-vercel-cli": "latest",
```

Comment:
Given this is a hacky thing and we're somewhat distrusting of it, I think pinning to a specific version would be good here. Let's us very precisely control the dependency on this fork, so we know exactly what version we're running of `@cloudflare/next-on-pages` and of the Vercel CLI fork.